shortcircuit-api/src/test/resources/config.yml

```diff
@@ -3,5 +3,12 @@ short-circuit-parameters:
   with-fortescue-result: true
   with-voltage-result: false
   study-type: "SUB_TRANSIENT"
+  sub-transient-coefficient: 0.8
   with-feeder-result: false
-  min-voltage-drop-proportional-threshold: 1.2
\ No newline at end of file
+  min-voltage-drop-proportional-threshold: 1.2
+  with-loads: true
+  with-shunt-compensators: true
+  with-vsc-converter-stations: false
+  with-neutral-position: true
+  initial-voltage-profile-mode: "CONFIGURED"
+  voltage-range-data: "voltage-range-data.json"
```

Comment:
Is it working with an absolute path or a relative path?

Reply:
I think that the JSON file should be in the same folder as the config file. So basically this field should just have the name of the file.

Reply:
With CGMES boundaries, we put the files in general in a folder inside de `.itools`. It should work for this too, in order to create a folder but for short-circuit.